indigo/distribution.yaml

```diff
@@ -10239,8 +10258,6 @@ repositories:
       version: indigo-devel
     release:
       packages:
-      - collada_parser
-      - collada_urdf
```

Comment:
Is it ensured that future releases of this repo don't try to re-release these two packages?